synapse/lib/stormtypes.py

```diff
@@ -619,8 +619,7 @@ async def _libDmonStop(self, iden):
         viewiden = ddef['stormopts']['view']
         self.runt.confirm(('dmon', 'add'), gateiden=viewiden)
 
-        await self.runt.snap.core.disableStormDmon(iden)
-        return True
+        return await self.runt.snap.core.disableStormDmon(iden)
```

Comment:
The docs state that this always returns 'True', any concern about backward compatibility here or should we update them since they can return 'False' now?

Reply:
Good catch.  I'd consider it a bug if the API was to return True when it didnt' do anything, so i'll update the docs.